docs/source/dataframe-joins.rst

```diff
@@ -33,12 +33,17 @@ parallel way, where each partition of the large DataFrame is joined against the
 single small table.  This incurs almost no overhead relative to Pandas joins.
 
 If your smaller table can easily fit in memory, then you might want to ensure
-that it is a single partition with the following
+that it is a single partition with the repartition method.
 
 .. code-block:: python
 
+    from dask.datasets import timeseries
```

Comment:
I mention this below, but it's more typical to `import dask` and then use timeseries like `dask.datasets.timeseries`